### ISSUE TITLE
Unable to use voiceover on devices running iOS 11 and above

### ISSUE THREAD

**Issue by fastroadrunner** [FokkeZB/nl.fokkezb.drawer#84]
When using the widget, voiceover on iOS 11 and up is not able to detect the elements on the screen. The elements are accessible when used on a device with iOS 10.
